app/Http/Controllers/DocumentsController.php

```
@@ -123,38 +123,35 @@ public function upload(Request $request, $external_id)
         $client = Client::whereExternalId($external_id)->first();
 
         $file        = $request->file('file');
-        $filename    = str_random(8) . '_' . $file->getClientOriginalName();
+        $filename    = Str::random(8) . '_' . $file->getClientOriginalName();
         $fileOrginal = $file->getClientOriginalName();
 
         $size       = $file->getSize();
         $mbsize     = $size / 1048576;
         $totaltsize = mb_substr($mbsize, 0, 4);
```

PHPStan (2.2.5)

[warning] Parameter `#1` $string of function mb_substr expects string, (float|int) given.


(argument.type)